overlord/snapstate/backend_test.go

```diff
@@ -1372,7 +1405,7 @@ func (f *fakeSnappyBackend) RemoveSnapDir(s snap.PlaceInfo, otherInstances bool)
 	f.appendOp(&fakeOp{
 		op:             "remove-snap-dir",
 		name:           s.InstanceName(),
-		path:           snap.BaseDir(s.SnapName()),
+		path:           snap.BaseDir(s.InstanceName()),
```

Comment:
This change here I think is better representative of what actually happens in `RemoveSnapDataDir` (the real implementation).